django/utils/encoding.py

```diff
@@ -53,7 +53,7 @@ def force_str(s, encoding='utf-8', strings_only=False, errors='strict'):
     If strings_only is True, don't convert (some) non-string-like objects.
     """
     # Handle the common case first for performance reasons.
-    if issubclass(type(s), str):
+    if isinstance(s, str):
         return s
     if strings_only and is_protected_type(s):
         return s
```

Comment:
note that https://github.com/django/django/pull/11375/files#diff-d3fe562051dcbab80f60f067fa6a5838R61 uses isinstance. Should it be `issubclass(type(s`?